great-idea/css/index.css

```diff
@@ -63,4 +63,140 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* Your code starts here! */
+nav {
+	/* border: 1px solid red; */
+	width: 1000px;
+	/* padding: 15px; */
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+nav a {
+	color: grey;
+	text-decoration: none;
+	/* margin-right: 50px; */
+}
+
+h1.home {
+	font-size: 75px;
+}
+
+h1.services-h1 {
+	font-size: 40px;
+}
+
+ul {
+	list-style-type: square;
+	padding: 20px;
+	line-height: 2;
+}
+
+button {
+	border: 1px solid black;
+	padding: 5px 30px;
+}
+
+button.services-button {
+	border-radius: 5px;
+	margin-top: 15px;
+}
+
+footer {
+	text-align: center;
+}
+
+.container {
```

Comment:
You may want to move this to the top. Keep in mind your CSS should flow in mirror to your HTML for easier reading.